chart/values.yaml

```diff
@@ -8,6 +8,7 @@ image:
 gitImage: alpine/git:latest
 busyboxImage: busybox:latest
 terraformImage: oamdev/docker-terraform:1.1.2
+jobNamespace: ""
```

Comment:
How about changing it to `ControllerNamespace`? Most users, maybe, don't know the underlying workload (Job) to executing Terraform.

Reply:
Yep, makes more seens - I've changed across the board (command line, controller var and meta var to ControllerNamespace)